lstchain/mc/sensitivity.py

```diff
@@ -272,15 +272,11 @@ def calculate_sensitivity_lima(n_on_events, n_background, alpha):
         n_off=n_background,
         alpha=alpha
         )
-
-
     n_excesses_5sigma = stat.excess_matching_significance(5)
-    for value_excesses, value_bkg in np.nditer([n_excesses_5sigma, n_background], op_flags=['readwrite']):
-        if value_excesses < 10:
-            value_excesses[...] = 10
-        if value_excesses < 0.05 * value_bkg * alpha[0]:
-            value_excesses[...]=0.05 * value_bkg * alpha[0]
-        
+    n_excesses_5sigma[n_excesses_5sigma<10] = 10
```

Comment:
Way do you replace smaller values with 10? That artificially introduces signal that is not there.

Reply:
same here